app/src/main/java/com/uc/mymovies/MainActivity.kt

```diff
@@ -64,7 +64,8 @@ class MainActivity : AppCompatActivity() {
 
             override fun afterTextChanged(searchTxt: Editable?) {
                 if (searchTxt?.isNotEmpty() == true) {
-                    var filteredQuery = testMovieData.filter { movie -> movie.title.toLowerCase().contains(searchTxt) };
+                    var treatedSearchTxt = searchTxt.toString().toLowerCase()
+                    var filteredQuery = testMovieData.filter { movie -> movie.title.toLowerCase().contains(treatedSearchTxt) };
```

Comment:
When I initially opened the project and put "Avatar" in the search bar, I got no results back. This is because you are searching through the movie titles after turning all the titles to all lower case. Without also turning the search text into all lowercase, any upper case letters will mess with the filtering. to keep consistent with the movie title string, I also made the search text all lowercase.

Reply:
Great catch!